connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -207,3 +208,23 @@ message Sample {
     int64 seed = 1;
   }
 }
+
+// Relation of type [[Range]] that generates a sequence of integers.
+message Range {
+  // Optional. Default value = 0
+  int32 start = 1;
+  int32 end = 2;
```

Comment:
end is not optional, but how do we know if the client forgets to set it? 0 is a valid end value as well.

Reply:
Yeah this becomes tricky. Ultimately we can wrap every such field into a message so we always know if that field is set or not set. However that might complicate entire proto too much.. Let's have a discussion on that.